src/server.js

```diff
@@ -173,6 +185,15 @@ function endsWithExtensions(file, extensions) {
   return false;
 }
 
+function getFileFromType(track, type) {
```

Comment:
This function should probably have a doc comment explaining that e.g. if not found you get "none" and not `null` or something falsey.